### PR TITLE
docs(en): add doubly linked list

### PR DESCRIPTION
**Pull request by Ayebilla**
**PR Checklist:**
- [x] My submission is formatted according to the guidelines in the [contributing guide](https://github.com/MakeContributions/DSA#contribution-guidelines)
- [x] My addition is on refer on the language README.md file
- [x] My addition does not have a spelling problem
- [x] My submission has a proper and user-friendly description of the algorithm
- [x] My submission has the time complexity of the algorithm
- [ ] My submission has sample input-output of the program (NOT FOR PYTHON)

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] New algorithm
- [ ] Optimization in previous algorithms
- [ ] Code style update
- [ ] Refactor
- [x] Documentation
- [ ] Other, please describe:

**Briefly describe the changes in this PR**
Add docs(en) for doubly linked list. 

**Other information:**
 Addresses #docs(en): doubly-linked-list
Please, review.

### CODE REVIEW

File: docs/en/Lists/doubly-linked-list.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+- [Python](../../../algorithms/Python/linked_lists/doubly.py)
+
+## Video URL
+
+[Doubly linked list](https://www.youtube.com/watch?v=nquQ_fYGGA4)
+
+
+## References
+
+Goodrich, T. M., Tamassia, R., Goldwasser, M.H.(2014). *Data Structures & Algorithms in Java* (6th ed.). Wiley
```
Comment (ming-tsai) [inline docs/en/Lists/doubly-linked-list.md:68]: Do you have a link for this reference?

File: docs/en/Lists/doubly-linked-list.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
+# Doubly Linked List
+
+A doubly linked list is a type of linked data structure that allows traversal in both directions (forward and backward directions). It differs from singly linked list in that, singly linked list does not support backward traversal. Every node in a doubly linked list contains a **data** part and two pointers; **next pointer** that points to the next node and the **prev pointer** that points to the previous node in the list. A doubly linked list also has two dummy nodes (head and tail) known as **sentinels** (guard nodes) that contains no data of the sequence. The first element of a doubly linked list has its prev pointer pointing to head and the last element has its next node pointing to tail. 
+
+Without a tail pointer, you cannot directly access the last element without having to first traverse the entire list all the way from the beginning. 
+![doubly linked list](https://github.com/Ayebilla/project/blob/main/doubly-linked-list.PNG?raw=true)
```
Comment (ming-tsai) [inline docs/en/Lists/doubly-linked-list.md:6]: The icon doesn't work, and please remove it.

File: docs/en/Lists/doubly-linked-list.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
+# Doubly Linked List
+
+A doubly linked list is a type of linked data structure that allows traversal in both directions (forward and backward directions). It differs from singly linked list in that, singly linked list does not support backward traversal. Every node in a doubly linked list contains a **data** part and two pointers; **next pointer** that points to the next node and the **prev pointer** that points to the previous node in the list. A doubly linked list also has two dummy nodes (head and tail) known as **sentinels** (guard nodes) that contains no data of the sequence. The first element of a doubly linked list has its prev pointer pointing to head and the last element has its next node pointing to tail.
```
Comment (ming-tsai) [inline docs/en/Lists/doubly-linked-list.md:3]: Could you add a new line? due to someone doesn't wrap the word on his editor.

**Review by ming-tsai — CHANGES_REQUESTED** [review]
Hi @Ayebilla,
Some suggestions for this pull request.

File: docs/en/Lists/doubly-linked-list.md (hunk 3 above)
Comment (Ayebilla) [reply docs/en/Lists/doubly-linked-list.md]: Thank you for the suggestions, I have added the link now.

**Comment by Ayebilla** [comment]
The icon has been removed as well. That edit was mistakenly wrapped in the first commit

**Comment by Ayebilla** [comment]
Thank you @ming-tsai for taking time to review my pull request. I have implemented the suggested changes and waiting for your approval.

**Review by ming-tsai — APPROVED** [review]

**Pull request merged**